ssz_serialization/proofs.nim

```diff
@@ -229,8 +224,18 @@ func calculate_multi_merkle_root*(
   if leaves.len != indices.len:
     return err("Length mismatch for leaves and indices")
   ? check_multiproof_acceptable(indices)
-  calculate_multi_merkle_root_impl(
-    leaves, proof, indices, get_helper_indices(indices))
+  let helper_indices = get_helper_indices(indices)
+  calculate_multi_merkle_root_impl(leaves, proof, indices, helper_indices)
+
+func calculate_multi_merkle_root*(
+    leaves: openArray[Digest],
+    proof: openArray[Digest],
+    indices: static openArray[GeneralizedIndex]): Result[Digest, string] =
+  if leaves.len != indices.len:
+    return err("Length mismatch for leaves and indices")
+  static: ? check_multiproof_acceptable(indices)
```

Comment:
The `?` operator is not well defined in static context. It translates to a `return` statement propagating the error.

Reply:
That is the intended behaviour here. The function would just statically become one that always tosses an error.